hydra-test-utils/hydra-test-utils.cabal

```diff
@@ -54,9 +66,31 @@ library
     , temporary
     , warp
 
-  default-extensions:
-    NoImplicitPrelude
-    FlexibleContexts
-
   ghc-options:        -Wall -Werror -Wcompat -Wunused-packages
   default-language:   Haskell2010
+
+test-suite hydra-test-util-tests
+  import: package-config
+  hs-source-dirs:     test
+  other-modules:
+    Test.HSpec.MarkdownFormatterSpec
+    HydraTestUtilsSpec
+
+  main-is:            Main.hs
+  type:               exitcode-stdio-1.0
+  build-depends:
+                      base
+                , QuickCheck
+                , bytestring
+                , directory
+                , filepath
+                , hspec
+                , hspec-core
+                , hydra-prelude
+                , hydra-test-utils
+                , temporary
+                , text
+                , text
```

Comment:
Could: fix the indentation, it is inconsistent with the remaining file (use `cabal-fmt` to format `.cabal` files)